Project.toml

```
@@ -12,7 +13,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 [compat]
 ChainRulesCore = "0.6.1"
 ChainRulesTestUtils = "0.1.3"
```

Needs bumping to match https://github.com/JuliaDiff/ChainRulesTestUtils.jl/pull/17

```suggestion
ChainRulesTestUtils = "0.2"
```